src/test/groovy/org/codenarc/idea/testing/GroovyProjectDescriptors.java

```diff
@@ -15,6 +15,7 @@ public interface GroovyProjectDescriptors {
   TestLibrary LIB_GROOVY_2_4 = new RepositoryTestLibrary("org.codehaus.groovy:groovy-all:2.4.17");
   TestLibrary LIB_GROOVY_2_5 = new RepositoryTestLibrary("org.codehaus.groovy:groovy:2.5.12");
   TestLibrary LIB_GROOVY_3_0 = new RepositoryTestLibrary("org.codehaus.groovy:groovy:3.0.7");
+  TestLibrary LIB_GROOVY_4_0 = new RepositoryTestLibrary("org.apache.groovy:groovy:4.0.10");
```

Comment:
this seem unused

Reply:
It seems the entire file is almost entirely unused. I have added so that i think it makes sense, but perhaps i should delete instead.